di_website/templates/downloads/download-list.html

```diff
@@ -19,34 +19,35 @@ <h2 class="type-m type-s--trailer">{{ page.publication_downloads_title }}</h2>
                 {% endfor %}
             </ul>
         {% endif %}
-        {% if page.data_downloads_list or page.filtered_datasets %}
```

Comment:
hmm ... by removing this check, the title on line 23 and any other conditional markup will always be shown even when there's no downloads ... 
This is the result:
![image](https://user-images.githubusercontent.com/5672438/85831228-a76ae780-b796-11ea-9009-e424ee408850.png)

And since this is a reused template, this affects about 6 types of pages. Dejavu!